chrysom/README.md

```diff
@@ -0,0 +1,36 @@
+# chrysom
+The client library for communicating with Argus over HTTP.
+[![PkgGoDev](https://pkg.go.dev/github.com/xmidt-org/argus@v0.5.1/chrysom)](https://pkg.go.dev/github.com/xmidt-org/argus@v0.5.1/chrysom)
```

Comment:
```suggestion

[![Go Reference](https://pkg.go.dev/badge/github.com/xmidt-org/argus/chrysom.svg)](https://pkg.go.dev/github.com/xmidt-org/argus/chrysom)
```

The badge looked broken...I tried to manually fix things, but by all accounts what you did looks right.  I got this suggestion from this page:
https://pkg.go.dev/badge/

I took out the version so we won't have to update this badge when we make a new release.